src/modules/economy/components/DepositForm/useCreateDepositAPI.ts

```diff
@@ -14,6 +15,20 @@ export function useCreateDepositAPI() {
       variables: {
         input: input,
       },
+      onCompleted() {
+        showNotification({
+          title: 'Suksess',
+          message: 'Innskudd har blitt opprettet',
+          color: 'Green',
+        })
+        onCompletedCallback()
```

Comment:
We're a bit inconsistent with usage of success notifications. Do we want them to just be a green notification with "success" or should we show a success icon instead?